examples/compiled/errorband_2d_horizontal_color_encoding.vg.json

```diff
@@ -39,7 +39,7 @@
           "orient": {"value": "vertical"},
           "fill": {"value": "black"},
           "tooltip": {
-            "signal": "{\"Miles per Gallon (95% CIs)\": format(datum[\"lower_Miles_per_Gallon\"], \"\"), \"upper_Miles_per_Gallon\": format(datum[\"upper_Miles_per_Gallon\"], \"\"), \"Year (year)\": timeFormat(datum[\"year_Year\"], '%Y')}"
+            "signal": "{\"Ci1 of Miles_per_Gallon\": format(datum[\"upper_Miles_per_Gallon\"], \"\"), \"Ci0 of Miles_per_Gallon\": format(datum[\"lower_Miles_per_Gallon\"], \"\"), \"Year (year)\": timeFormat(datum[\"year_Year\"], '%Y')}"
```

Comment:
Why "Ci" instead of "CI"? It stands for Confidence Interval.

Reply:
As commented above, we should systematically revise title of ci0/ci1 for both composite mark and general operation in a separate PR.

Reply:
Okay, I'll let you make the call on this or then.